packages/components/atoms/f-filter-pill/src/components/FilterPill.vue

```diff
@@ -105,8 +105,10 @@ export default {
 </script>
 
 <style lang="scss" module>
-$filter-pill-selected: $color-content-positive;
-$filter-pill-checkbox-width: 16px;
+$selected-color: $color-content-positive;
+$checkbox-width: 16px;
+$transition-duration: 0.2s;
+$ease: ease-in-out;
 
```

Comment:
I think its good to prefix these variables with the component name like you had before - maybe we can check the fozzie convention but potentially these could be: `$filter-pill-selected-color`, `$filter-pill-ease` ...

Reply:
yeah, I'm not sure if we have the convention. The variables won't leak into other components, so I don't really see much sense in making them longer and harder to read. Happy to change if we have such guidelines.

Reply:
For sure, I've taken a look at the other atoms and a couple of the molecules in Fozzie and they generally follow this pattern but up to you whether you want to change or not 👍

Reply:
Changed it to follow the pattern. I'm still not sure if consistency is more important than readability, but let's see how it goes, and adjust along the way.